### ISSUE TITLE
Sync user display with logged in user

### ISSUE THREAD

**Issue by jb3rndt** [team-folivora/annoto#89]
What the title says

**Comment by n-koenig** [comment]
![Image](https://user-images.githubusercontent.com/30596454/166696712-d65f2790-83c1-4807-a64a-18262c6f5952.png)

Isn't that already done? Or do you mean something else?

EDIT: I'm stupid, it's still hardcoded :) Should be done after #78 is finished
